daprdocs/content/en/operations/monitoring/newrelic.md

```diff
@@ -0,0 +1,347 @@
+
+## Automated installer
+
+1. Select the New Relic account you want to send events, metrics and logs to.
+2. Provide a name of your AKS cluster
+3. Provide the namespace for Kubernetes integration
+4. Select the details of the integration
+5. Download the Kubernetes manifest file
+6. Note: If you plan to automate the setup process – with Helm chart, for instance – continue in [with docs](https://docs.newrelic.com/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm)
+
+![New Relic Kubernetes Automated Installer](/images/nr-automated-installer.png)
+
+## Deploy integration
+
+To deploy, run this command in your console, and insert the path to where you downloaded the manifest.
+
+```bash
+kubectl apply -f newrelic-manifest.yaml -n dapr-monitoring
+```
+
+## Validation
+
+### Ensure New Relic Kubernetes integration is running.
+
+```bash
+kubectl get pods -n dapr-monitoring
+NAME                                            READY STATUS  RESTARTS  AGE
+nri-bundle-kube-state-metrics-f7cf766ff-2bdt6   1/1   Running 0         16m
+nri-bundle-newrelic-infrastructure-48wdk        1/1   Running 0         16m
+nri-bundle-newrelic-infrastructure-fj4pz        1/1   Running 0         16m
+nri-bundle-newrelic-logging-7gw8k               1/1   Running 0         16m
+nri-bundle-newrelic-logging-b6fwx               1/1   Running 0         16m
+nri-bundle-nri-kube-events-6899758c57-rfmvq     2/2   Running 0         16m
+nri-bundle-nri-prometheus-689d7b6f45-gfrgx      1/1   Running 0         16m
+```
+
+### Ensure Dapr is running
+
+```bash
+kubectl get pods -n dapr-system
+NAME                                READY STATUS  RESTARTS  AGE
+dapr-operator-59ddb59f4c-4fgzm      1/1   Running 0         16m
+dapr-placement-599d87d5cb-n6zpc     1/1   Running 0         16m
+dapr-sentry-777658c665-z4qhb        1/1   Running 0         16m
+dapr-sidecar-injector-56c4588-4rs8z 1/1   Running 0         16m
+```
+
+## New Relic Kubernetes Cluster Explorer
+
+The [New Relic Kubernetes Cluster Explorer](https://docs.newrelic.com/docs/integrations/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer) provides a unique visualization of the entire data and deployments of the data collected by the Kubernetes integration.
+
+![New Relic Kubernetes Cluster Explorer](/images/nr-k8s-cluster-explorer.png)
+
+## Install Quickstart application Distributed calculator
```

Comment:
I believe this section can be removed. Setting up the calculator is not a required step to enable New Relic for Dapr. This documentation should ideally only include New Relic specific instructions and Dapr configuration - the application part should be left to the user